tools/docker/big-env/Dockerfile

```diff
@@ -42,6 +42,9 @@ RUN dpkg --add-architecture i386 && \
 	apt-get clean autoclean && \
 	rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
 
+RUN rm -rf /usr/local/go
+RUN curl https://dl.google.com/go/go1.19.6.linux-amd64.tar.gz | tar -C /usr/local -xz
```

Comment:
We also need at least `ENV PATH /usr/local/go/bin:/gopath/bin:$PATH` with this.

Reply:
It is already configured in docker/env. And we reuse that container here.